src/settings/CommentRequiredForm.js

```diff
@@ -45,7 +45,7 @@ const Setting = ({
 );
 
 Setting.propTypes = {
-  label: PropTypes.element,
+  label: PropTypes.string,
```

Comment:
Hey Zak,
I see the reason underlying in here (Select accepts aria-label prop of type string), that made you update label prop type as string.
Wouldn't eslint complaint in this case? How did it go unaddressed till now?

Reply:
ESLint is only going to do static analysis so it won't pick up this kind of thing.